source/slang/slang-ir-insts-info.cpp

```diff
@@ -0,0 +1,63 @@
+#include "core/slang-basic.h"
+#include "slang-ir.h"
+
+namespace Slang
+{
+
+struct IROpMapEntry
+{
+    IROp op;
+    IROpInfo info;
+};
+
+// TODO: We should ideally be speeding up the name->inst
+// mapping by using a dictionary, or even by pre-computing
+// a hash table to be stored as a `static const` array.
+//
+// NOTE! That this array is now constructed in such a way that looking up
+// an entry from an op is fast, by keeping blocks of main, and pseudo ops in same order
+// as the ops themselves. Care must be taken to keep this constraint.
+static const IROpMapEntry kIROps[] = {
+
+// Main ops in order
+
+#if 0 // FIDDLE TEMPLATE:
+% RAW(require("source/slang/slang-ir.h.lua").instInfoEntries())
```

Comment:
Same questions here:

* Why does this need to use `RAW`?
* Why can't this just define the code directly, rather than needing to chase down that subroutine?